src/components/DatePickerComponent.tsx

```diff
@@ -8,7 +8,7 @@ import { InlineComponentProps } from "../CruxComponent"
 export class DatePickerComponent extends React.Component<InlineComponentProps, any> {
     constructor(props: any) {
         super(props)
-        this.state = {}
+        this.state = { interval: 30 }
```

Comment:
this change is backward compatible right ?

Reply:
Yeah it is driven through config. It will be showed only when field has showTimeSelect

Reply:
Does this require documentation in readme ?